spinalcordtoolbox/reports/slice.py

```diff
@@ -270,6 +278,26 @@ def mosaic(self, nb_column=0, size=15, return_center=False):
         else:
             return matrices
 
+    def mosaics_through_time(self):
+        """Obtain mosaics for each volume
+
+        :return: list of tuples of numpy.ndarray containing the mosaics of each volumes
+        """
+
+        mosaics = list()
+        self._image_seg = self._images[0].copy()  # segmentation used for cropping
+
+        for i, img in enumerate(self._4d_images):
+            # The absolutepath is changed to None after change_orientation see issue #3304
+            img.absolutepath = self._absolute_paths[i]
```

Comment:
Adding a link here to https://github.com/neuropoly/spinalcordtoolbox/issues/3304 to remind us to revisit.